tools/mmp/driver.cs

```diff
@@ -1177,7 +1177,7 @@ static int Compile (IEnumerable<string> internalSymbols)
 				break;
 			case "x86_64":
 				if (IsClassic)
-					throw new MonoMacException (5204, true, "Invalid architecture. x86_64 is only supported with the mobile profile.");
+					throw new MonoMacException (5204, true, "Invalid architecture. x86_64 is only supported with the Modern (Mobile) profile.");
```

Comment:
The message (even the old one) is wrong. We don't support 64bits on classic but other XM profiles (Full or Modern) do support 64 bits.